capplets/accessibility/at-properties/main.c

```diff
@@ -23,43 +25,56 @@ enum {
 
 static gboolean initial_state;
 
+static GDesktopAppInfo *get_desktop_app_info_from_dm (void)
+{
+    DMType dm_type;
+    GDesktopAppInfo *app_info = NULL;
+
+    dm_type = dm_get_type();
+    if (dm_type == DM_TYPE_MDM) {
+        app_info = g_desktop_app_info_new ("mdmsetup.desktop");
```

Comment:
As mentioned before, mdmsetup support can be dropped...

Reply:
@yetist 
Any chance to remove `mdmsetup` too?
Than we can finish this PR.